utm_tracker/request.py

```diff
@@ -1,23 +1,28 @@
-from typing import List
-
 from django.http import HttpRequest
 
+from .types import UtmParamsDict
 
-def request_has_utm_params(
-    request: HttpRequest, required_params: List[str] = None
-) -> bool:
-    """
-    Return True if the request has UTM paramaters we consider valid.
+VALID_UTM_PARAMS = [
+    "utm_source",
+    "utm_medium",
+    "utm_campaign",
+    "utm_term",
+    "utm_content",
+]
 
-    This function should be called before automatically persisting a
-    LeadSource, to ensure that one is only persisted when enough valid
-    UTM parameters exist for it to make sense.
 
-    The `required_params` parameter can be overriden on a per-call basis to
-    decide which parameters should be required for a given call. We provide
-    a sane default.
+def parse_qs(request: HttpRequest) -> UtmParamsDict:
     """
-    required_params = required_params or ["utm_source", "utm_medium"]
+    Extract 'utm_*' values from request querystring.
 
-    required_values = [request.session.get(param, "") for param in required_params]
-    return all(required_values)
+    NB in the case where there are multiple values for the same key,
```

Comment:
This rename will also break platform usage in one place, but I think it can just be removed in that place now that this library is clever enough not to store an empty UtmDict by default.